strawberryfields/backends/fockbackend/ops.py

```diff
@@ -185,7 +186,7 @@ def project_reset(modes, x, state, pure, n, trunc):
 
     def intersperse(lst):
         # pylint: disable=missing-docstring
-        return tuple([lst[i // 2] for i in range(len(lst) * 2)])
+        return tuple(lst[i // 2] for i in range(len(lst) * 2))
```

Comment:
nice catch!

Reply:
just noticed this. isn't this more readable? `tuple([x for el in lst for x in (el,el)])`

Reply:
isn't that less performant as it has two loops?

Reply:
I don't think the difference is even noticeable